docs/content/mobile/ios/setup.md

```diff
@@ -2,13 +2,18 @@
 
 ## Get the dependencies for developing iOS games
 
+> **Note:** iOS development is only supported on macOS.
```

Comment:
well... technically, you can build it on Windows via Unity. However, the resulting XCode project needs to be built using a mac

Reply:
agreed with @jessicafalk that while this is not strictly true as there are workarounds, it's worth being brief here than getting into the weeds on the issue in an overview